thrift/atoms/tenfour.thrift

```diff
@@ -0,0 +1,61 @@
+namespace * tenfour
+
+include "../shared.thrift"
+
+/** date times are reprsented as i64 - epoch millis */
+typedef i64 DateTime
+
+struct Content {
+  // do we need to store the ID, seeing as it is replicated(?) in the
+  // content-atom wrapping?
```

Comment:
It might be usefull if we want to be able to link form a quiz in prod through to the editor with teleporter...